interview-questions.md

```diff
@@ -14,44 +14,45 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is a version control software used on local computer. GitHub is version control software cloud based.
 
-Researched answer:
+Researched answer: Git is a version control system that allows you to track source code history. GitHub is cloud based service that allows you to manage repositories and keep track of them. 
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: Logical operators
 
-Researched answer:
+Researched answer: Logical operators AND, OR and  NOT
```

Comment:
Make sure to expand further in the researched answer.  The bare minimum here is one complete sentence.